haproxy/templates/configmap-galerak8s.yaml

```
@@ -0,0 +1,45 @@
+{{- if eq .Values.haproxy.config "galerak8s"}}
+{{- $galera := .Values.haproxy.galerak8s -}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: "{{ include "haproxy.fullname" . }}-galerak8s"
+  labels:
+    app.kubernetes.io/name: {{ include "haproxy.name" . }}
+    helm.sh/chart: {{ include "haproxy.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+data:
+  haproxy.cfg: |-
+    global
+      log stdout local0
+      stats socket :9000 mode 660 level admin
+
+    
+    resolvers mydns
+      nameserver dns1 {{ $galera.nameserver }}
```

```suggestion
      parse-resolv-conf
```

This should add all nameservers from the `/etc/resolv.conf` file in the pod and therefore uses the cluster default DNS: https://cbonte.github.io/haproxy-dconv/2.2/configuration.html#5.3.2-parse-resolv-conf